video/out/w32_common.c

```diff
@@ -953,6 +953,26 @@ static bool is_visible(HWND window)
     return GetWindowLongPtrW(window, GWL_STYLE) & WS_VISIBLE;
 }
 
+//Set the mpv window's affinity.
+//This will affect how it's displayed on the desktop and in system-level operations like taking screenshots.
+//During startup of mpv we want to avoid changing anything when its not needed, that is toggled by handleDefault
+static bool update_affinity(struct vo_w32_state *w32, bool handleDefault)
+{
+    bool affinity_result = false;
+    if (!w32 || w32->parent) {
+        return affinity_result;
+    }
+    if ((w32->opts->window_affinity >= 1) || (handleDefault)) {
+        // Note: On versions prior to Win10 WDA_EXCLUDEFROMCAPTURE will internally be using WDA_MONITOR.
+        affinity_result = SetWindowDisplayAffinity(w32->window, 
+            w32->opts->window_affinity == 1 ? WDA_EXCLUDEFROMCAPTURE :
+            w32->opts->window_affinity == 2 ? WDA_MONITOR :
+            WDA_NONE
```

Comment:
You can set `WDA_` values directly in `OPT_CHOICE` to void this mapping.

Reply:
Changed it to use OPT_CHOICE for the mapping now, although I must've messed up the git push/merge..